e2e/vm/additional_disk_test.go

```diff
@@ -12,32 +14,46 @@ import (
 
 const (
 	savedImage    = "public.ecr.aws/docker/library/alpine:latest"
-	containerName = "userDataTest"
+	containerName = "test-ctr"
+	volumeName    = "test-volume"
+	networkName   = "test-network"
 )
 
 var testAdditionalDisk = func(o *option.Option) {
 	ginkgo.Describe("Additional disk", ginkgo.Serial, func() {
 		ginkgo.It("Retains container user data after the VM is deleted", func() {
-			command.Run(o, "pull", savedImage)
-			ginkgo.DeferCleanup(command.Run, o, "rmi", savedImage)
-			oldImagesOutput := command.StdoutStr(o, "images", "--format", "{{.Name}}")
-			gomega.Expect(oldImagesOutput).Should(gomega.ContainSubstring(savedImage))
+			command.Run(o, "volume", "create", volumeName)
+			ginkgo.DeferCleanup(command.Run, o, "volume", "rm", volumeName)
+			command.Run(o, "network", "create", networkName)
+			ginkgo.DeferCleanup(command.Run, o, "network", "rm", networkName)
 
-			command.Run(o, "run", "--name", containerName, savedImage)
-			ginkgo.DeferCleanup(command.Run, o, "rm", containerName)
-			oldPsOutput := command.StdoutStr(o, "ps", "--all", "--format", "{{.Names}}")
-			gomega.Expect(oldPsOutput).Should(gomega.ContainSubstring(containerName))
+			command.Run(o, "run", "-d", "--name", containerName, "-v", fmt.Sprintf("%s:/tmp", volumeName),
+				savedImage, "sh", "-c", "echo foo > /tmp/test.txt; sleep infinity")
+			ginkgo.DeferCleanup(command.Run, o, "rmi", savedImage)
+			ginkgo.DeferCleanup(command.Run, o, "rm", "-f", containerName)
 
-			command.New(o, virtualMachineRootCmd, "stop").WithoutCheckingExitCode().WithTimeoutInSeconds(90).Run()
+			command.New(o, virtualMachineRootCmd, "stop").WithoutCheckingExitCode().WithTimeoutInSeconds(180).Run()
```

Comment:
in my experience, the `vm stop` command tends to take longer if there is a running container, perhaps we could do `finch kill test-ctr` instead of increasing the timeout here?